### ISSUE TITLE
Opt-in to python 3.8

### ISSUE THREAD

**Issue by deathaxe** [facelessuser/FuzzyFileNav#42]
FuzzyFileNav runs perfectly well on python 3.8 plugin host.

How about officially migrating it?

**Comment by facelessuser** [comment]
Hmm, I never added dependencies to this one. All I have to do is drop a python version specifying file in it?

**Comment by deathaxe** [comment]
Yes, a `.python-version` file with `3.8` is all which is needed.

**Comment by facelessuser** [comment]
Cool, that shouldn't be a problem.
